client/homepage/style.scss

```diff
@@ -13,18 +13,41 @@
 		width: calc(50% - 12px);
 		margin: 0;
 
-		@include breakpoint('<782px') {
+		@include breakpoint( '<782px' ) {
 			width: 100%;
 			position: inherit;
 			top: auto;
 		}
+
+		& > * {
+			margin-bottom: 24px;
+		}
 	}
 
-	@include breakpoint('<782px') {
+	@include breakpoint( '<782px' ) {
 		margin-left: -16px;
 		margin-right: -16px;
 		flex-direction: column-reverse;
 	}
+
+	.components-card {
+		.components-card__header.is-size-large {
```

Comment:
Are these rules going to apply to all large cards on the home page or just this one? If just this one, maybe something specific to the quicklinks would be best.

Reply:
I'd expect we'd want to apply it to all large cards on the home page, for consistency.

Reply:
Further, I'd expect that all cards on the home page should use the `@wordpress/components` `Card`.

Reply:
Cool, I saw that issue 👍